.github/workflows/Android-CI.yml

```diff
@@ -0,0 +1,33 @@
+name: PullRequest
+
+on: [pull_request]
+
+jobs:
+  buildTest:
+    name: Build SDK and HelloCardboard
+    runs-on: macOS-latest
```

Comment:
Let's use either `macOS` for everything.

Also, there are several versions of macOS available. Make sure to stick to macOS 10.15--> https://github.com/actions/virtual-environments/blob/macOS-10.15/20210314.1/images/macos/macos-10.15-Readme.md